### ISSUE TITLE
Audio streams no longer work

### ISSUE THREAD

**Issue by An-anonymous-coder** [FossifyOrg/Calendar#394]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Calendar/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Calendar/issues) or [closed](https://github.com/FossifyOrg/Calendar/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Calendar/discussions) or [closed](https://github.com/FossifyOrg/Calendar/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Calendar/blob/master/CONTRIBUTING.md).

### Affected app version

1.1.0

### Affected Android/Custom ROM version

GrapheneOS 2024120900

### Affected device model

Pixel 8

### How did you install the app?

GitHub releases

### Steps to reproduce the bug

1. Open Fossify Calendar
2. Tap on the 3 dots on the top right
3. Tap on `Settings`
4. Under `REMINDERS`, tap on `Audio stream used by reminders`
5. Set it to `Alarm` or `Ring`
6. Go back
7. Create an event or task with a reminder that will happen in a timely manner
8. Close Fossify Calendar
9. Open Settings
10. Tap `Sound & vibration`
11. Set `Ring volume` and `Alarm volume` to maximum
12. Set `Notification volume` to minimum
13. Wait for the event reminder to trigger

### Expected behavior

The notification should use the `Ring` or `Alarm` audio streams. 

### Actual behavior

The notification will only vibrate. It will not use the `Alarm` or `Ring` audio streams. 

### Additional information

This only started I think after GrapheneOS 2024120702. It was working fine before.

**Comment by ReaderGuy42** [comment]
I have the same issue.

**Comment by Aga-C** [comment]
@ReaderGuy42 What device model and OS version do you have?

**Comment by ReaderGuy42** [comment]
Pixel 6a, GrapheneOS 2024120900

**Comment by Aga-C** [comment]
@An-anonymous-coder We haven't investigated it further, so I can't tell right now if it's GrapheneOS bug (I couldn't reproduce it on other Androids), or we don't support properly some feature of it. You can raise the bug on their side as well.

**Comment by ReaderGuy42** [comment]
There has since been an update, to build 2024121200 with no change. I raised a bug on the GrapheneOS forum before commenting here, they pointed me here. So I'm not sure what to do.

**Comment by ReaderGuy42** [comment]
Is this something the user can change? Or are you asking the developers?

**Comment by ReaderGuy42** [comment]
Any news  on this?

**Comment by naveensingh** [comment]
None yet.

**Comment by naveensingh** [comment]
@An-anonymous-coder @ReaderGuy42 

Here's a test APK: https://github.com/FossifyOrg/Calendar/actions/runs/16261914183/artifacts/3524789241

**Comment by ReaderGuy42** [comment]
@naveensingh Thanks! Can I install this side by side with the regular app?

**Comment by naveensingh** [comment]
Yes, but I don't recommend it. It's a debug build. The official release will be out soon.

Thanks for testing!

**Comment by user-1-2-3-4-5** [comment]
At least the 'system'  and 'alarm' streams don't play audio on my LineageOS 23 (Android 16) device. I maxed out the alarm volume and only the notifications were created for a handful of test events, all of which certainly had the reminder enabled. I don't use the do not disturb mode. For audible alarms I frequently use both the pre-installed alarm app alongside [RemindMe](https://github.com/VishnuSanal/RemindMe/).

I followed the reproduction steps and also disabled the Android battery optimization for Calendar (set to 'unrestricted'). Version 1.8.1 [from F-Droid](https://f-droid.org/en/packages/org.fossify.calendar/) on Oneplus 8T.

**Comment by naveensingh** [comment]
If anyone else has this issue, please comment.
